flixel/system/frontEnds/DebuggerFrontEnd.hx

```diff
@@ -35,6 +35,10 @@ class DebuggerFrontEnd
 	 * Dispatched when drawDebug is changed.
 	 */
 	public var drawDebugChanged(default, null):FlxSignal = new FlxSignal();
+	/**
+	 * Dispatched when visibility is changed.
+	 */
+	public var visibilityChanged(default, null):FlxTypedSignal<Bool->Void> = new FlxTypedSignal<Bool->Void>();
```

Comment:
No, it's not necessary, I was just trying to make things easier for the signal's user. I can remove the new value for the sake of consistency.